src/protocols/rdp/channels/cliprdr.c

```diff
@@ -180,8 +180,13 @@ static UINT guac_rdp_cliprdr_monitor_ready(CliprdrClientContext* cliprdr,
         return status;
 
     /* ... and supported format list */
-    return guac_rdp_cliprdr_send_format_list(cliprdr);
+    status = guac_rdp_cliprdr_send_format_list(cliprdr);
+    if (status != CHANNEL_RC_OK)
+        return status;
+
+    guac_common_clipboard_operational_send(clipboard->client);
```

Comment:
At this point the virtual channel is initialised end to end https://docs.microsoft.com/en-us/openspecs/windows_protocols/ms-rdpeclip/a5cae3c9-170c-4154-992d-9ac8a149cc7e